db/migrate/20220114110920_update_order_cycle_mails.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateOrderCycleMails < ActiveRecord::Migration[6.1]
+  class MigrationOrderCycle < ActiveRecord::Base
+    self.table_name = "order_cycles"
+  end
```

Comment:
Why not just `OrderCycle` and omit the table name?

I found that there's usually no conflict because the full class name here is actually `UpdateOrderCycleMails::OrderCycle`. That just doesn't work with polymorphic types.